src/main/java/org/folio/circulation/domain/Item.java

```diff
@@ -350,6 +351,13 @@ public boolean doesNotHaveHolding() {
     return holdingRepresentation == null;
   }
 
+  public String getPermanentLocationId() {
+    final String itemLocation = getProperty(itemRepresentation, PERMANENT_LOCATION_ID);
+    final String holdingsLocation = getProperty(holdingRepresentation, PERMANENT_LOCATION_ID);
+
+    return StringUtils.firstNonBlank(itemLocation, holdingsLocation);
```

Comment:
Personally, I might be tempted to statically import `firstNonBlank`
```suggestion
    return firstNonBlank(itemLocation, holdingsLocation);
```